parameters/password.parameters.json

```diff
@@ -1,13 +1,39 @@
 {
-	"_artifactsLocation":{"value":"https://raw.githubusercontent.com/chobbs/ARM-Templates/master/src/"},
-	"loadBalancerType":{"value":"external"},
-	"chronograf":{"value":"Yes"},
-	"vmSizeDataNodes":{"value":"Standard_DS2_v2"},
-	"vmDataNodeCount":{"value":2},
-	"vmDataDiskSize":{"value":"32GiB"},
-	"vmSizeMetaNodes":{"value":"Standard_D1_v2"},
-	"password":{"value":{"password":"Password1234","sshPublicKey":null,"authenticationType":"password"}},
-	"adminUsername":{"value":"influxadmin"},
-	"influxdbPassword":{"value":"changeME"},
-	"location":{"value":"centralus"}
-}
\ No newline at end of file
+    "_artifactsLocation": {
+        "value": "https://raw.githubusercontent.com/influxdata/azure-resource-manager-influxdb-enterprise/master/src/"
+    },
+    "loadBalancerType": {
+        "value": "external"
+    },
+    "chronograf": {
+        "value": "Yes"
+    },
+    "vmSizeDataNodes": {
+        "value": "Standard_DS2_v2"
+    },
+    "vmDataNodeCount": {
+        "value": 2
+    },
+    "vmDataDiskSize": {
+        "value": "32GiB"
+    },
+    "vmSizeMetaNodes": {
+        "value": "Standard_D1_v2"
+    },
+    "password": {
+        "value": {
+            "password": "change-me!",
```

Comment:
This default password string failed to parse and the template didn't load. Need to include include if the following: one upper case, one lower cases, one special character and one number.

suggestion: changeMe!4

Reply:
🤦‍♂ I clearly didn't test this change. I'll use your suggestion. Although I wish there was a way to have Azure automatically generate a secure password when creating the deployment. We should also add [support for Azure key vault](https://github.com/Azure/azure-quickstart-templates/tree/master/101-vm-secure-password) at some point.